src/utils/kenticoVersions.ts

```diff
@@ -1,5 +1,5 @@
-import MarketplaceItemModel from "@/models/marketplaceItemModel";
-import store, { updateKenticoVersionsFilterMutation } from "@/store";
+import MarketplaceItemModel from "./../models/MarketplaceItemModel";
```

Comment:
Why to use relative path ant not continue using [name module mapper](https://github.com/Kentico/ems-extension-marketplace/blob/master/package.json#L88) ?